readme.md

```diff
@@ -6,6 +6,13 @@
 > Process execution for humans
 
 
+---
+
+One of the maintainers [@ehmicky](https://github.com/ehmicky) is looking for a remote full-time position as a Node.js back-end lead. Feel free to [contact him](https://www.mickael-hebert.com)!
```

Comment:
I would link directly to the contact form:

```suggestion
One of the maintainers [@ehmicky](https://github.com/ehmicky) is looking for a remote full-time position as a Node.js back-end lead. Feel free to [contact him](https://www.mickael-hebert.com#contact)!
```

Reply:
I actually would like them to check out my website before contacting me. That's because I get many requests for roles that do not match (not remote, not Node.js/JavaScript or not back-end).

Reply:
Alright